chromium_src/components/permissions/permission_context_base.h

```diff
@@ -78,10 +78,11 @@ class PermissionContextBase : public PermissionContextBase_ChromiumImpl {
     BrowserPermissionCallback GetNextCallback();
     void RequestFinished();
 
-   private:
     std::vector<std::pair<std::unique_ptr<PermissionRequest>,
                           BrowserPermissionCallback>>
         requests_;
```

Comment:
class members cannot be public.

Reply:
fixed in https://github.com/brave/brave-core/pull/17439/commits/aa7c8e2de55cdf3d10bd4520253c8422046fb29e, thanks.